assets/static/js/dif.js

```diff
@@ -265,15 +262,27 @@ $(document).ready(function()
     });
 
     $.ajaxSetup({
-        error: function(x, t, m) {
-            var message;
-            if (typeof m.message != "undefined") {
-                message = m.message;}else{message = m;
-            }
-            if ((x.status == 400 || x.status == 403) && x.responseJSON) {
-                message = x.responseJSON.message;
+        error: function(jqXHR, textStatus, errorThrown) {
+            loadingSpinner.hideSpinner();
+            let message = "Server is Unreachable, please try again later!";
+            if (jqXHR.status !== 0) {
+                message = jqXHR.responseText == null ? errorThrown: jqXHR.responseJSON.message;
             }
-            console.log("Error in Ajax:"+t+", Message:"+message)
+            console.log("Error in Ajax:" + textStatus + ", Message:" + message);
+            var n = noty(
+                {
+                    layout: "top",
+                    theme: "relax",
+                    type: "error",
+                    text: message,
+                    modal: false,
+                    timeout: 2000,
```

Comment:
Maybe make this a bit longer...

Reply:
Can remove this noty? and change it error dialog, so that all of it is consistent? Or I could do it in my branch? What do you guys think